### DISCUSSION TITLE
Versions of SMB

### DISCUSSION THREAD

**Discussion by brianereynolds** [kubernetes-csi/csi-driver-smb#426] [Q&A] [answered]
Hi, very simple question, is something as old as SMB 1.0 supported? Thanks

**Comment by andyzhangx** [comment, marked as answer]
@brianereynolds this driver supports  2.1, 3.0, 3.1, and I have not test 1.0, I think you could have a try.

**Reply by andyzhangx** [reply]
and just let me know the result, thanks.
